azure/azure-how-to/instances/create-a-golden-image-pipeline.rst

```diff
@@ -0,0 +1,274 @@
+Create an Ubuntu golden image pipeline
+======================================
+
+This guide will provide instructions for using GitHub Actions to create a pipeline for building Ubuntu "golden" images with the Azure Image Builder (AIB). This guide focuses on automating the use of Azure Image Builder. If you want more information on how to use Azure Image Builder see the :doc:`Create a Pro golden image <create-pro-fips-golden-image>` guide which explains the individual steps.
+
+What you'll need
```

Comment:
To  give the feel of a sequence of steps to follow, can we change the section headings to something like:

- What you'll need
- Set up your Azure credentials
- Set up your GitHub repository
- --- Main executables
- ++++++ create_all_resources
- ++++++ run_image_builder
- --- Local setup
- Configure your golden image
- Configure your GitHub Action
- Accept the image terms
- Run the GitHub Action